src/UserInterface/index.js

```diff
@@ -8,6 +8,7 @@ import createPointSetsUI from './createPointSetsUI'
 import emptyContainer from './emptyContainer'
 import getRootContainer from './getRootContainer'
 import checkForWebGL from './checkForWebGL'
+import { updateSliceActors, updateVolumeProps } from './updateActors'
```

Comment:
Let's start organizing the rendering-related logic in `src/Rendering/`, so these functions can be exposed in `src/Rendering/index.js` instead of `src/UserInterface`.